.gitlab/e2e.yml

```diff
@@ -129,3 +129,24 @@ k8s-e2e-otlp-main:
   #   - dev_master-a7
   script:
     - inv -e e2e-tests --agent-image=datadog/agent-dev:master-py3 --dca-image=datadog/cluster-agent-dev:master --argo-workflow=otlp
+
+.new_e2e_template:
+  stage: e2e
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:a58cc31c
+  tags: ["arch:amd64"]
+  before_script:
+  # Setup AWS Credentials
+    - mkdir -p ~/.aws
+    - aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.agent-qa-profile --with-decryption --query "Parameter.Value" --out text >> ~/.aws/config
```

Comment:
Are there secrets in that entry? If yes, we'll want to surround this with `set +x` / `set -x` to avoid accidentally logging them.

Reply:
Should I add the `set -x` in a `after_script` section ?

Reply:
I see, the wrapper should only around this line, or we would miss test logs